Style-guide/README.md

```diff
@@ -71,8 +68,8 @@ Descriptive and consistent naming makes software easier to read and understand.
 
 - striving for clarity at the call site
 - prioritizing clarity over brevity
-- using camel case (not snake case)
-- using uppercase for types (and protocols), lowercase for everything else
+- using `camelCase` (not `snake_case`)
+- using uppercase initials for types (and protocols), lowercase first-initial for everything else
 - including all needed words while omitting needless words
 - using names based on roles, not types
 - sometimes compensating for weak type information
```

Comment:
This isn't clear to me. Is it to anyone else?

Reply:
I would add an example, maybe with something like `tvShowID: String`